Python/ceval_gil.c

```diff
@@ -1,6 +1,6 @@
 
 #include "Python.h"
-#include "pycore_atomic.h"        // _Py_atomic_int
+#include "pycore_atomic.h"        // _Py_ANNOTATE_RWLOCK_XXXX
```

Comment:
I would prefer to mention a real symbol:

```suggestion
#include "pycore_atomic.h"        // _Py_ANNOTATE_RWLOCK_CREATE
```

Same remark in Python/thread_pthread.h.